### DISCUSSION TITLE
How to install the latest version in ubuntu?

### DISCUSSION THREAD

**Discussion by aarcangeli** [ccache/ccache#1057] [Questions & answers] [answered]
Hi,

I'm trying to install ccache on ubuntu using this:

```
sudo apt update
sudo apt install -y ccache
```

however I cannot install the latest version (4.6).

```
$ ccache --version
ccache version 3.7.7
[...]
$ apt show ccache
Package: ccache
Version: 3.7.7-1
[...]
```

Looking in the release page (https://github.com/ccache/ccache/releases/tag/v4.6), I can only find windows binaries.

Is there a way to get linux binaries without compiling it from the source code?

Thanks in advance

**Comment by cristianadam** [comment]
I have binaries at https://github.com/cristianadam/ccache/releases/tag/v4.6 

That's the only reason I have the fork nowadays, just for the binaries so that I can use them on GitHub Actions.

**Reply by aarcangeli** [reply]
Thank you, really appreciated.

ccache should provide binaries for Linux as well.

**Comment by jrosdahl** [comment, marked as answer]
As of ccache 4.6.1 there is a Linux x86_64 binary release as well: https://ccache.dev/download.html

**Comment by alfred-ai** [comment]
To make it painfully obvious (I'm using Ubuntu 20.04):
1. `sudo apt remove ccache` (if you have the version came from the distribution installed)
1. Expand the downloaded archive file
   1. `cd` to it
   2. `sudo make install`
